### PR TITLE
New Hampshire

### PR DESCRIPTION
**Pull request by askpatrickw**
Motto: Live Free or Die !

New Hampshire was named by Captain John Mason after Hampshire, England. One of the original 13 colonies, New Hampshire became the 9th state on June 21, 1788.

### CODE REVIEW

File: states/new_hampshire/elections.toml [hunk 1]
```diff
@@ -3,7 +3,52 @@ date = 2020-09-08
 name = "Congressional Primary"
 original_date = 2020-09-08
 
+[20200908.registration]
+in_person_by = 2020-09-08
+sources = ["http://sos.nh.gov/WorkArea/DownloadAsset.aspx?id=8589993085"]
+
+# [20200908.absentee.application]
+# # The Mailed-In or dropped off early Ballot is the Application
```
Comment (tannewt) [inline states/new_hampshire/elections.toml]: Looks like you need to request being sent a ballot: http://sos.nh.gov/WorkArea/DownloadAsset.aspx?id=8589999336

From: http://sos.nh.gov/VotingEmergency.aspx

**Review by tannewt — CHANGES_REQUESTED** [review]
Kinda wild there is no registration besides in person. I believe @jerryneedell is in NH and can confirm.

File: states/new_hampshire/elections.toml (hunk 1 above)
Comment (askpatrickw) [reply states/new_hampshire/elections.toml]: You find all the things !! ;-)

Comment (jerryneedell) [reply states/new_hampshire/elections.toml]: Looks like you found the best source - do you have all the information you need for NH?

Comment (jerryneedell) [reply states/new_hampshire/elections.toml]: The key thing in NH this year is that concern about COVID-19 is considered a valid "disability" so you are permitted to use the "Absentee for Disability" forms for registration and for requesting an Absentee Ballot.

This is not a permanent change to the law, just a special allowance for the pandemic.
Normally absentee methods are only permitted for limited reasons.

Comment (tannewt) [reply states/new_hampshire/elections.toml]: Thanks @jerryneedell. We're doing these dates on an election-by-election basis for that reason. I think we could better convey it though going forwards.

**Review by tannewt — APPROVED** [review]
Thank you!

**Pull request merged**